tests/kernel/install_ltp.pm

```diff
@@ -344,6 +366,10 @@ sub run {
     else {
         add_ltp_repo;
         install_from_repo();
+        if (get_var("LTP_INSTALL_SELECTED")) {
```

Comment:
LGTM, but I think there's no need to add a new variable. It'd be better to check `LTP_GIT_URL` instead.

Reply:
Ideally, it'd be better to build the modules from LTP source package for version stability reasons but this is good enough for now.

Reply:
Ok changed to `LTP_GIT_URL`. This problem is only for for tests using daily ltp package builds, so fixed version is not needed.